File: Data Structures & Algorithms/concatenation-of-array/submission-4.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [" " for i in range(2 * (len(nums)))]
        for index, num in enumerate(nums):
            ans[index] = num
            ans[index + len(nums)] = num
        return ans

File: Data Structures & Algorithms/sort-colors/submission-4.java
class Solution {
    public void sortColors(int[] nums) {
        int[] count = new int[3];
        for (int i : nums) {
            count[i] += 1;
        }
        int index = 0;
        for (int i = 0; i < 3; i++) {
            while (count[i]-- > 0){
                nums[index++] = i;
}
        }

    }
}